clients/python/girder_client/cli.py

```diff
@@ -163,6 +177,16 @@ def main(ctx, username, password, api_key, api_url, scheme, host, port, api_root
     input his/her password.
     """
     # --api-url and URL by part arguments are mutually exclusive
+    if config:
+        ctx.params.update(_parseConfigFile(config))
+        username = ctx.params.get('username')
+        password = ctx.params.get('password')
+        api_key = ctx.params.get('api_key')
+        api_url = ctx.params.get('api_url')
+        scheme = ctx.params.get('scheme')
+        host = ctx.params.get('host')
+        port = ctx.params.get('port')
+        api_root = ctx.params.get('api_root')
```

Comment:
If someone passes `--config` but also passes one of these options explicitly, shouldn't the explicit option override the config file one?

Reply:
That sounds like useful behavior for the user. I'll do that.